_data/StemsAnalyzer.yml

```diff
@@ -0,0 +1,9 @@
+description: An extension for precision point insertion.
+developer: Jérémie Hornus and Connor Davenport
+developerURL: https://black-foundry.com https://connordavenport.com
```

Comment:
can you change this to a single url? maybe pointing to this readme of your extension repo where you mentions both urls?